protocols/autonatv2/src/client/handler.rs

```diff
@@ -7,397 +7,25 @@
 // TODO: Handlers
 
 mod dial_back;
+mod request;
 
-use std::{
-    cmp::min,
-    collections::VecDeque,
-    convert::identity,
-    io,
-    sync::Arc,
-    task::Poll,
-    time::Duration,
-};
-
-use futures::{AsyncRead, AsyncWrite, AsyncWriteExt};
-use libp2p_core::{upgrade::ReadyUpgrade, Multiaddr};
-use libp2p_identity::PeerId;
-use libp2p_swarm::{
-    handler::{ConnectionEvent, FullyNegotiatedInbound, FullyNegotiatedOutbound, ProtocolsChange},
-    ConnectionHandler, ConnectionHandlerEvent, StreamProtocol, SubstreamProtocol,
-};
+use std::time::Duration;
 
+use dial_back::Handler as DialBackHandler;
```

Comment:
I'd much prefer if we use `dial_back::Handler` throughout the code instead renaming upon import. It is less code (no imports), equally expressive and comparable in length.